src/enum_net.cpp

```diff
@@ -701,7 +622,6 @@ int _System __libsocket_sysctl(int* mib, u_int namelen, void *oldp, size_t *oldl
 
 					// ignore errors here. This is best-effort
 					ioctl(s, siocgifmtu, &req);
```

Comment:
I can clear out the remaining MTU stuff. I feel bad for "feature creeping" your PR